pycode/memilio-simulation/CMakeLists.txt

```diff
@@ -42,8 +42,8 @@ function(add_pymio_module target_name)
 
     pybind11_add_module(${target_name} MODULE ${PYBIND11_MODULE_SOURCES})
     target_link_libraries(${target_name} PRIVATE ${PYBIND11_MODULE_LINKED_LIBRARIES})
-    target_include_directories(${target_name} PRIVATE memilio/simulation)
-    install(TARGETS ${target_name} LIBRARY DESTINATION memilio/simulation/bindings)
+    target_include_directories(${target_name} PRIVATE memilio/simulation/bindings)
+    install(TARGETS ${target_name} LIBRARY DESTINATION memilio/simulation)
```

Comment:
that change is ok, but I don't think this fixes the issue. these commands are mostly declarative and order doesn't usually matter. 

I think the path is just wrong, the CMakeLists.txt is inside the memilio directory, so the path should just be `simulation/bindings`. Or maybe better `${CMAKE_CURRENT_SOURCE_DIR}/simulation/bindings` to be explicit.

Reply:
it wasnt a change of order. The install path was set to the bindings folder, instead of the include directory

Reply:
I can't read :shrug:. you are right, that's certainly a bug. 

still don't think this causes the build include errors since it should only affect the install step afterwards.

Reply:
Dont we need the includes of header files during the build process? And those are all relative paths expecting that memilio/simulation/bindings is know to the compiler as a include directory. Else they pointing nowhere. And before the include directory was set wrong.